feature-libs/product-configurator/rulebased/core/state/actions/configurator.action.ts

```diff
@@ -170,7 +170,7 @@ export class ChangeGroup extends StateUtils.EntityLoadAction {
     public payload: {
       configuration: Configurator.Configuration;
       groupId: string;
-      parentGroupId: string;
+      parentGroupId?: string;
```

Comment:
What does it mean when parentGroupId is undefined? Perhaps you could explain in a JSDoc?

Reply:
Yes, will do. It means the group is on root level